Whiskey/Tasks/Version.ps1

```diff
@@ -91,6 +100,33 @@ function Set-WhiskeyVersion
         $rawVersion = $Version
         $semVer = $rawVersion | ConvertTo-SemVer -PropertyName 'Version'
     }
+    elseif ($DateFormat)
+    {
+        $rawVersion = Get-Date -Format $DateFormat

```

Comment:
Any reason we should have the task base the version on the datetime that the build started at? As opposed to getting the datetime the moment the Version task executes.